components/layout/blog-homepage-section/blog-homepage-section.hbs

```diff
@@ -1,168 +1,171 @@
 <div class="section--bg-white">
-    <div class="content-block--homepage-blog">
-
-        <!-- Featured Blog -->
-        <a href="#" class="content-block--clickable content-block--blog-featured-home">
-
-            <div class="content-block__upper-blog">
-                <picture class="post-image">
-                    <source media="(max-width: 799px)" srcset="https://www.fillmurray.com/g/600/634">
-                    <source media="(min-width: 800px)" srcset="https://www.fillmurray.com/g/525/720">
-                    <img class="content-block__image" src="https://www.fillmurray.com/g/545/747" role="presentation"
-                        alt="" />
-                    <source>
-                </picture>
-            </div><!-- .content-block__upper-blog -->
-
-            <div class="content-block__lower-blog">
-                <h3 class="content-block__title anchor-primary">
-                    Using Your UX Skills to Architect Your Career
-                </h3>
-
-                <div class="content-block__blog-author-date">
-                    <label for="source" class="content-block__author">By Mags Hanley</label>
-                    <time class="content-block__blog-date" datetime="2021-06-10"><span
-                            class="content-block__blog-date-month-day-year" aria-label="June">06.02.21</span></time>
+    <div class="section__inner">
+        <div class="content-block--homepage-blog">
```

Comment:
This doesn't need to be two divs (unless you need it to be for your own layout purposes). You can just add `section__inner` to your content block classList and it will apply the spacing as necessary.

Reply:
Updated.